docs/configuration.asciidoc

```diff
@@ -108,6 +108,26 @@ NOTE: The service name must conform to this regular expression: ^[a-zA-Z0-9 _-]+
 | Name of the entry assembly       | String 
 |============
 
+[float]
+[[config-service-version]]
+==== `ServiceVersion`
+
+This is used to know exact version of your service.
+
+NOTE: The service version must conform to this regular expression: ^[0-9]+\.[0-9]+(?:\.[0-9]+)?(?:\.[0-9]+)?$. In less regexy terms: Your service version must contain only numbers (from 2 to 4), which are separated by `.` (dot).
```

Comment:
Could you please clarify where these constrains are coming from? I don't see any constrains neither in [other agents documentation](https://docs.google.com/spreadsheets/d/1JJjZotapacA3FkHc2sv_0wiChILi3uKnkwLTjtBmxwU/) nor in [APM server API specification](https://github.com/elastic/apm-server/blob/6.5/docs/spec/service.json#L79).

Reply:
Ouch. I didn't look into documentation. My fault, will fix it.

Reply:
Ouch. I didn't look into documentation. My fault, will fix it.